src/features/projects/ManageLabelsModal/EditLabelForm.jsx

```diff
@@ -0,0 +1,64 @@
+import { useCallback, useState } from "react";
+import { useDispatch } from "react-redux";
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+
+import { updateProjectLabel } from "../../filters/filtersSlice.js";
+import LabelPill from "../../../components/LabelPill";
+import IconButton from '../../../components/IconButton.jsx';
+import { Pencil1Icon } from '@radix-ui/react-icons';
+import LabelForm from './LabelForm';
+import {
+  LabelRow,
+  LabelHeader,
+  LabelActions,
+} from './components';
+
+export const label = Yup.object().shape({
+  name: Yup.string().required('Enter a label name.'),
```

Comment:
@oliverroick thanks for doing this, but I'm noticing that you can no longer edit labels (say if you just edit the color) because that dupe checking runs when you try to save it again, and it fails b/c obviously there's already a label in there with that name.

Reply:
Different issue, but I'm also noticing that if you add multiple labels in a row, each time you click New Label, the form is populated with the values from the previous label, rather than the default vals.

Reply:
@oliverroick I hopped on these and am addressing them this AM so no action needed on your end.